### DISCUSSION TITLE
$sort pipeline with Typescript: Type 'number' is not assignable to type '1 | -1 | { $meta: "textScore"; }'

### DISCUSSION THREAD

**Discussion by raj-kumar94** [Automattic/mongoose#11153] [Q&A] [answered]
Hi, I'm trying to sort using aggregation pipeline

```
{
    $sort: {
        created_at: -1
    }
},
```

but getting the following error

`Type 'number' is not assignable to type '1 | -1 | { $meta: "textScore"; }'`

Sorry if this is a newbie question. It was working fine earlier. I'm using version `6.1.4`

Thanks in advance.

**Comment by matvienkont** [comment, marked as answer]
Possible workaround:
https://github.com/Automattic/mongoose/issues/11169#issuecomment-1030167841

**Reply by raj-kumar94** [reply]
Thanks, it works.
